airbyte-integrations/connectors/source-rss/integration_tests/catalog.json

```diff
@@ -0,0 +1,39 @@
+{
+  "streams": [
+    {
+      "name": "TODO fix this file",
```

Comment:
@jrhizor can you please fix this TODO

Reply:
Done.

Just curious: Where is this actually used? it wasn't failing when I ran `python -m pytest integration_tests -p integration_tests.acceptance` so I assume the discovery acceptance test wasn't actually comparing with this, and everything else is using one of the configured catalog files.